app/helpers/era_av_helper.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017, University of Alberta Libraries.
```

Comment:
I know this is what Avalon does but do we want to follow this behaviour of having a copyright notice on every single file of code in this codebase that you create (also considering it's a Apache/MIT license too)? Especially when there is a LICENSE file in the root of the project too that explains all this?